README.md

```diff
@@ -1,31 +1,113 @@
-# nebari
+<div align="center">
+ <img alt="Nebari logo on white background" src="./docs/static/img/docusaurus.png" width="300" />
+</div>
+<br>
 
+| Information | Links |
+| :---------- | :-----|
+|   Project   | [![License](https://img.shields.io/badge/License-BSD%203--Clause-gray.svg?colorA=2D2A56&colorB=5936D9&style=flat.svg)](https://opensource.org/licenses/BSD-3-Clause) [![Nebari documentation](https://img.shields.io/badge/%F0%9F%93%96%20Read-the%20docs-gray.svg?colorA=2D2A56&colorB=5936D9&style=flat.svg)](https://nebari-docs.netlify.app/) [![PyPI version](https://badge.fury.io/py/qhub.svg)](https://badge.fury.io/py/qhub) |
+|  Community  | [![GH discussions](https://img.shields.io/badge/%F0%9F%92%AC%20-Participate%20in%20discussions-gray.svg?colorA=2D2A56&colorB=5936D9&style=flat.svg)](https://github.com/nebari-dev/nebari/discussions) [![Open an issue](https://img.shields.io/badge/%F0%9F%93%9D%20Open-an%20issue-gray.svg?colorA=2D2A56&colorB=5936D9&style=flat.svg)](https://github.com/nebari-dev/nebari/issues/new/choose) |
+|     CI      | [![Kubernetes Tests](https://github.com/Quansight/qhub/actions/workflows/kubernetes_test.yaml/badge.svg)](https://github.com/Quansight/qhub/actions/workflows/kubernetes_test.yaml) [![Tests](https://github.com/Quansight/qhub/actions/workflows/test.yaml/badge.svg)](https://github.com/Quansight/qhub/actions/workflows/test.yaml) |
+| Website | [![Website Deploy](https://github.com/nebari-dev/nebari/actions/workflows/deploy.yml/badge.svg)](https://github.com/nebari-dev/nebari/actions/workflows/deploy.yml) 
 
-🏗 An opinionated open source JupyterHub distribution
+---
 
-## Documentation
+Nebari is an customizable Data Science and MLOPs Platform designed as a managed integration of open source technologies. 
```

Comment:
```suggestion
Nebari is a customizable Data Science and MLOPs platform designed as a managed integration of open source technologies. 
```

Reply:
I know this is the wording we carried over from the old repo - there is something that does not sound entirely right to me but cannot tell what it is. 
Will need to revisit later

Reply:
its so generic that it almost has no meaning

Reply:
yes - my brain is not super creative today so will come back to it tomorrow